examples/ogbn_papers_100m.py

```diff
@@ -119,17 +212,68 @@ def test(loader: NeighborLoader, val_steps: Optional[int] = None):
     return total_correct / total_examples
 
 
-torch.cuda.synchronize()
+test_accs = []
+val_accs = []
+times = []
+train_times = []
+inference_times = []
+best_val = best_test = 0.
 prep_time = round(time.perf_counter() - wall_clock_start, 2)
 print("Total time before training begins (prep_time)=", prep_time, "seconds")
 print("Beginning training...")
-for epoch in range(1, 1 + args.epochs):
-    train()
-    val_acc = test(val_loader, val_steps=100)
-    print(f'Val Acc: ~{val_acc:.4f}')
+for run in range(1, num_runs + 1):
```

Comment:
I'm in favour of removing this for-loop because it'll simplify the script, and users can easily run the script multiple times.